tooling/debugger/src/context.rs

```diff
@@ -78,6 +78,21 @@ impl<'a, B: BlackBoxFunctionSolver> DebugContext<'a, B> {
         }
     }
 
+    pub(super) fn get_call_stack(&self) -> Vec<OpcodeLocation> {
+        let ip = self.acvm.instruction_pointer();
```

Comment:
Nit but I think it is clearer when we don't use short names like this even though this method is quite small. 

```suggestion
        let instruction_pointer = self.acvm.instruction_pointer();
```

Reply:
Same thing for the new `get_call_stack` in the brillig_vm. Lets expand `v`